src/middlewared/middlewared/plugins/support.py

```diff
@@ -172,10 +169,9 @@ async def fetch_categories(self, token):
         'new_ticket',
         Str('title', required=True, max_length=None),
         Str('body', required=True, max_length=None),
-        Str('category', required=True),
         Bool('attach_debug', default=False),
+        Dict('debug_extra', additional_attrs=True),
         Password('token'),
-        Str('type', enum=['BUG', 'FEATURE']),
```

Comment:
Why is it removed? We use this field to allow users to choose between Bug or Feature they want to create:

<img width="296" alt="image" src="https://github.com/truenas/middleware/assets/351613/d6abc1e8-e86b-4b6f-b34b-5a3bc02dbfe1">

Reply:
@yocalebo we remove Bug/Feature distinction, don't we?